server/bin/unittests

```diff
@@ -257,6 +257,16 @@ function _verify_output {
     return ${res}
 }
 
+function _create_user {
+    name=${1}
+    # NOTE: we ignore failures here deliberately; it'll always fail in
+    # the 0[.x] tests because of intentionally corrupted configuration,
+    # and unintentional failure in other tests will result in visible
+    # symptoms later.
+    python3 ./lib/pbench/cli/server/user_create.py --username=${name} --email=${name}@example.com \
+        --first-name=${name^} --last-name=User --password=password >/dev/null 2>&1
```

Comment:
It was a cute little bash-ism for the user's "real name": it capitalizes the substitution!

```
dbutenho 16:44 templatedb:templatedb x="david"
dbutenho 17:03 templatedb:templatedb echo ${x^}
David
```

Reply:
I figured it was something like that.  B-)

I had to ask because it didn't work in the bash I have on OS X:
```
wscales-mac:pbench_upstream wscales$ foo=dave
wscales-mac:pbench_upstream wscales$ echo ${foo^}
-bash: ${foo^}: bad substitution
```
But, it works in my Centos-8 container.  :-D